examples/features/retry/client/main.go

```diff
@@ -47,7 +47,9 @@ var (
 		}]}`
 )
 
-// use grpc.WithDefaultServiceConfig() to set service config
+// use grpc.WithDefaultServiceConfig() to set service config. Ideal
```

Comment:
slight improvement: "However, the recommended approach 
is to fetch the retry configuration from the name resolver rather than defining it on the client side."

Reply:
I like purnesh' exact wording a bit more than what you ended up with.

Also, if this is a documentation string it goes above the function definition but it should start with `retryDial` as outlined in https://tip.golang.org/doc/comment, otherwise you shoud probably move it one line below, above `grpc.Dial`. While you're here, can you also replace `retryDial` with `newClientWithRetries` or something like that, since it doesn't use dial anymore?

Reply:
thAnks @atollena, @purnesh42H for the detailed review, i have updated all the suggestion.